contents/handbook/company/team/annika-schmid.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Annika Schmid's README
+sidebar: Handbook
+showTitle: true
+---
+
+## Bio
+See our [team page](/handbook/company/team)
+
+## Areas of focus
+
+I primarily work with the [Feature Success](/handbook/small-teams/feature-success) and [Monitoring](/handbook/small-teams/monitoring) teams.
+
+I have written up how I think I can have the highest impact in my role in Q2 2023 and beyond. Let me know if I can be helpful in any other way.
+
+**How does my best work look like in Q2:**
+
+- We have a continuous interview process, where we speak to a couple of users a week. I am managing the interviews, having the team members join and co-lead the session with questions relevant to what they are working on.
+- Through user research and industry research, I surface the biggest problem areas relevant to the teams and products I am working with, and collect all the relevant background research so that the teams can confidently prioritise and implement solutions. As required, I support the team with prioritisation and designing the right solution for the problem.
+- Because of my background in design, I can, if required, lead or co-lead designing new features, for example through conduction ideation sessions, wireframing concepts in Figma, and conducting usability tests. Every engineer has a different preference here, so I work closely with the feature owner to get to the desired outcome. I also ensure that our existing products stay usable and make sure we balance new feature development with iterating on existing features to increase their usage and usability.
```

Comment:
```suggestion
- Because of my background in design, I can, if required, lead or co-lead designing new features, for example through conducting ideation sessions, wireframing concepts in Figma, and conducting usability tests. Every engineer has a different preference here, so I work closely with the feature owner to get to the desired outcome. I also ensure that our existing products stay usable and make sure we balance new feature development with iterating on existing features to increase their usage and usability.
```